### ISSUE TITLE
Update Hungarian translations

### ISSUE THREAD

**Issue by tibitoth** [squidfunk/mkdocs-material#4929]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### Language availability

- [X] I've checked the [list of available languages](https://bit.ly/33vFDD0)

### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "hu",
  "action.edit": "Oldal szerkesztése",
  "action.skip": "Kihagyás",
  "action.view": "Oldal forrásának megtekintése",
  "announce.dismiss": "Ne mutasd többet",
  "blog.archive": "Archívum", // ⁉️ verb or noun?
  "blog.categories": "Kategóriák",
  "blog.categories.in": "in ⬅️", // ⁉️ need more context
  "blog.continue": "Folytatás",
  "blog.draft": "Piszkozat",
  "blog.index": "Vissza a főoldalra",
  "blog.meta": "Metaadat",
  "blog.references": "Kapcsolódó linkek",
  "clipboard.copy": "Másolás vágólapra",
  "clipboard.copied": "Vágólapra másolva",
  "consent.accept": "Elfogadás",
  "consent.manage": "Beállítások",
  "consent.reject": "Visszautasítás",
  "footer": "Élőláb",
  "footer.next": "Következő",
  "footer.previous": "Előző",
  "header": "Élőfej",
  "meta.comments": "Hozzászólások",
  "meta.source": "Forrás",
  "nav": "Navigáció",
  "readtime.one": "1 percnyi",
  "readtime.other": "# percnyi",
  "rss.created": "RSS feed",
  "rss.updated": "Frissített tartalom RSS feedje",
  "search": "Keresés",
  "search.placeholder": "Keresés",
  "search.share": "Megosztás",
  "search.reset": "Törlés",
  "search.result.initializer": "Keresés inicializálása",
  "search.result.placeholder": "Kereséshez írj ide valamit",
  "search.result.none": "Nincs találat",
  "search.result.one": "1 egyező dokumentum",
  "search.result.other": "# egyező dokumentum",
  "search.result.more.one": "1 további találat az oldalon",
  "search.result.more.other": "# további találat az oldalon",
  "search.result.term.missing": "Üres",
  "select.language": "Nyelvváltás",
  "select.version": "Verzióváltás",
  "source": "Főoldalra ugrás",
  "source.file.contributors": "Szerzők",
  "source.file.date.created": "Létrehozva",
  "source.file.date.updated": "Utolsó frissítés",
  "tabs": "Lapok",
  "toc": "Tartalomjegyzék",
  "top": "Vissza a tetejére"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks!

>   "blog.categories.in": "in ⬅️", // ⁉️ need more context

This is the placeholder that is used to render the blog categories a post is categorized in:

<img width="468" alt="Bildschirmfoto 2023-01-29 um 12 37 50" src="https://user-images.githubusercontent.com/932156/215323488-7e936a74-bc82-47a8-a799-8ab73e19321c.png">

**Comment by squidfunk** [comment]
Added in 664af4236. I left out `blog.categories.in` for now – we can add it in another commit 😊

**Comment by squidfunk** [comment]
Released as part of [9.0.8](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.8).
